tests/foreman/cli/test_fact.py

```diff
@@ -12,49 +12,41 @@
 
 :TestType: Functional
 
-:CaseImportance: High
+:CaseImportance: Critical
 
 :Upstream: No
 """
 import pytest
 from fauxfactory import gen_string
 
 from robottelo.cli.fact import Fact
-from robottelo.test import CLITestCase
 
 
-class FactTestCase(CLITestCase):
-    """Fact related tests."""
+pytestmark = [pytest.mark.tier1]
```

Comment:
@mshriver can you tell me how this work? I mean why it's here and why we don't need to mark test like the way we used to?

Reply:
https://docs.pytest.org/en/stable/reference.html#globalvar-pytestmark

Sure!  pytest will read the `pytestmark` variable from a test module and apply the listed mark decorators to all of the classes/functions within the module.

Just like the testimony tokens, the marks don't have to be repeated on every test.